components/UI/Drawer/Drawer.tsx

```diff
@@ -113,18 +121,43 @@ const Drawer = () => {
   }
 
   const { handleMarkerClick: toggler } = useMapClickHandlers();
-
+  const router = useRouter();
   const list = useMemo(() => {
     if (!data) {
       return null;
     }
 
     const { geometry, formatted_address, source } = data;
+
+    let extraParams: undefined | ExtraParameters;
+
+    try {
+      extraParams = JSON.parse(source.extra_parameters || "false");
+    } catch (_) {
```

Comment:
Bu sekilde de calisiyor olmasi lazim 

```suggestion
    } catch  {
```

Yalniz burada bir sikinti var. `JSON.parse("false")` `false donuyor, boolean. Ama `extraParams` type definition'a gore `undefined | ExtraParameters` olabiliyor, yani oradaki type break etmis oluyoruz aslinda. 

Bunu improve etme sansimiz var mi? False yerine `undefined` yapsak mesela? Boolean olmak zorunda mi?